easy-ui-react/src/Banner/Banner.module.scss

```diff
@@ -0,0 +1,69 @@
+@use "../styles/common" as *;
+
+.variantSuccess {
+  @include component-token(
+    "banner",
+    "color",
+    theme-token("color.text.success.inverse")
+  );
+  @include component-token(
+    "banner",
+    "background_color",
+    theme-token("color.background.success")
+  );
+}
+
+.variantPrimary,
+.variantNeutral {
+  @include component-token(
+    "banner",
+    "color",
+    theme-token("color.text.inverse")
+  );
+}
+
+.variantPrimary {
+  @include component-token(
+    "banner",
+    "background_color",
+    theme-token("color.background.primary")
+  );
+}
+
+.variantNeutral {
+  @include component-token(
+    "banner",
+    "background_color",
+    theme-token("color.background.neutral")
+  );
+}
+
+.Banner {
+  @include component-token(
+    "banner",
+    "border_radius",
+    design-token("shape.border_radius.base")
+  );
+  @include component-token("banner", "max-width", 1348px);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0;
+  flex-wrap: wrap;
+  white-space: pre-wrap;
+  max-width: component-token("banner", "max-width");
+  padding: design-token("space.1") design-token("space.2");
```

Comment:
will check with design about a few of these styling discrepancies: 

- padding numbers, the figma suggests `32px` horizontally on desktop, `16px` on tablet, and `24px` on mobile, but i imagine the latter two should be swapped.
- the colors for the `success` variant, i imagine its supposed to have `blue-800` text as is the case with the buttons, but it looks like `#000` in the spec. 
- overall height of the banner in the design spec suggests it should be `32px`, but i think thats intended to be `40px` unless im mistaken